terraform/trial_rg/modules/privateendpoint/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "private_link_endpoint_ip" {
```

Comment:
might want to name which private link this is, since you'll end up with some.

Reply:
akv will need one also

Reply:
this is a reusable module.
I call it twice....